uint8 HANDS_UP         = 0
uint8 HANDS_ON_HEAD    = 1
uint8 HANDS_ON_FRONT   = 2

uint8 pose
uint8 timeout
